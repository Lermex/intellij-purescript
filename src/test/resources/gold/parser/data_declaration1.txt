Purescript File
  PSProgramImpl(Program)
    PSModuleImpl(Module)
      PsiElement(module)('module')
      PsiElement(whitespace)(' ')
      PSProperNameImpl(ModuleName)
        PSProperNameImpl(Qualified)
          PsiElement(proper name)('A')
        PsiElement(whitespace)(' ')
      PsiElement(where)('where')
      PsiElement(whitespace)('\n\n')
      PSDataDeclarationImpl(DataDeclaration)
        PsiElement(data)('data')
        PsiElement(whitespace)(' ')
        PSIdentifierImpl(TypeConstructor)
          PSProperNameImpl(ProperName)
            PsiElement(proper name)('User1')
        PSTypeArgsImpl(TypeArgs)
          <empty list>