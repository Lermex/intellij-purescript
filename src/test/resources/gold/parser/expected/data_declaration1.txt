Purescript File
  PSASTWrapperElement(Program)
    PSASTWrapperElement(Module)
      PsiElement(module)('module')
      PsiElement(whitespace)(' ')
      PSASTWrapperElement(Qualified)
        PSASTWrapperElement(ModuleName)
          PsiElement(proper name)('A')
      PsiElement(whitespace)(' ')
      PsiElement(where)('where')
      PsiElement(whitespace)('\n\n')
      PSASTWrapperElement(DataDeclaration)
        PsiElement(data)('data')
        PsiElement(whitespace)(' ')
        PsiElement(proper name)('User1')
        PSASTWrapperElement(TypeArgs)
          <empty list>